src/other/PropertyEditor.js

```diff
@@ -74,6 +81,21 @@
         ;
         table
             .each(function (d) {
+                if (d instanceof PropertyEditor === false) {
+                    if (context.overrideTags() && Object.keys(context.overrideTags()).length > 0) {
+                        var proto = Object.getPrototypeOf(d);
+                        for (var key in context.overrideTags()) {
+                            if (d._class.indexOf(key) !== -1) {
+                                for (var param in context.overrideTags()[key]) {
+                                    if (proto["__meta_" + param].ext && proto["__meta_" + param].ext.tags) {
+                                        proto["__meta_" + param].ext.tags = context.overrideTags()[key][param];
```

Comment:
@mlzummo It seems that you're overriding the tags on publish parameters to allow for removal/inclusion of specific publish params in the PropertyEditor? It would be a better approach to have an array publish param of widget class + publish param name combinations for either inclusion/removal.